package.json

```diff
@@ -5,8 +5,8 @@
   "private": true,
   "type": "module",
   "scripts": {
-    "build": "nuxt build apps/ui",
-    "build_debug": "NODE_OPTIONS='--trace-deprecation' nuxt build apps/ui",
+    "build": "dotenv nuxt build apps/ui",
```

Comment:
Verify that the newly introduced `dotenv` command in the build scripts does not inadvertently expose sensitive environment variables during the build process. [important]